### PR TITLE
className prop for each fading element

### PR DESCRIPTION
**Pull request by Deveosys**
Hey, it's me again ! 

I propose to pass a className prop applied for each fading element.
Feel free to find a better name if you deem it necessary.

### CODE REVIEW

File: src/FadeIn.js [hunk 1]
```diff
@@ -35,6 +35,7 @@ export default class FadeIn extends Component {
         {React.Children.map(this.props.children, (child, i) => {
           return (
             <div
+              className={this.props.itemClassName}
```
Comment (gkaemmer) [inline src/FadeIn.js:38]: Maybe call it `childClassName`?

**Review by gkaemmer — CHANGES_REQUESTED** [review]

**Comment by Henriii** [comment]
Just commenting to let you know I'd also request this feature, or any other way to manipulate the style of the animated/fading divs.

**Comment by gkaemmer** [comment]
Closing in favor of #7 -- thanks for a good idea! I'll give you credit for this change in the changelog.

**Comment by gkaemmer** [comment]
This has been released in `0.1.8` as `childClassName`

**Pull request closed without merging**